manim/mobject/functions.py

```diff
@@ -119,12 +119,12 @@ def generate_points(self):
 
 
 class FunctionGraph(ParametricFunction):
-    def __init__(self, function, color=YELLOW, **kwargs):
-        self.x_min = -config["frame_x_radius"]
-        self.x_max = config["frame_x_radius"]
+    def __init__(self, function, x_min=None, x_max=None, **kwargs):
```

Comment:
As I understand it, different mobjects are intended to have different default colors to add more variety and such. So, I think the `color` argument should be added back

Reply:
Yeah, that's true. Thanks for the review, I'll update the PR.